src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -143,35 +143,40 @@ const SubmissionDetails = _.flow(
     ])
   }
 
+  /**
+   * If the text has multiple lines, return just the first line of text with ellipses to indicate truncation.
+   */
+  const firstLine = text => {
+    const lines = text ? text.split('\n') : []
+    return lines.length > 1 ? `${lines[0]} ...` : lines.length > 0 ? lines[0] : undefined
+  }
+
   /*
    * Page render
    */
   return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
     submissionDetailsBreadcrumbSubtitle(namespace, name, submissionId),
     _.isEmpty(submission) ? centeredSpinner() : h(Fragment, [
       div({ style: { display: 'grid', gridTemplateColumns: '1fr 4fr' } }, [
-        div({ style: { display: 'grid', gridTemplateRows: 'repeat(3, 33%)' } }, [
+        div({ style: { display: 'grid', gridAutoRows: 'min-content' } }, [
           makeSection('Workflow Statuses', [
             succeeded && makeStatusLine(successIcon, `Succeeded: ${succeeded.length}`, { marginTop: '0.5rem' }),
-            failed && makeStatusLine(failedIcon, `Failed: ${failed.length}`, { marginTop: '0.5rem', marginBottom: '0.5rem' }),
+            failed && makeStatusLine(failedIcon, `Failed: ${failed.length}`, { marginTop: '0.5rem' }),
             running && makeStatusLine(runningIcon, `Running: ${running.length}`, { marginTop: '0.5rem' }),
             submitted && makeStatusLine(submittedIcon, `Submitted: ${submitted.length}`, { marginTop: '0.5rem' })
           ]),
           div({
-            style: {
-              padding: '0 0.5rem 0.5rem', marginTop: '1rem',
-              whiteSpace: 'pre', overflow: 'hidden', gridRow: '2 / 4'
-            }
+            style: { padding: '0 0.5rem 0.5rem', marginTop: '0.5rem', whiteSpace: 'pre', overflow: 'hidden' }
           }, [
             h4({ style: Style.elements.sectionHeader }, [
               'Comment',
               h(ButtonSecondary, {
-                style: { marginLeft: '0.50em' },
+                style: { marginLeft: '0.50em', height: '1.0em' },
```

Comment:
Reducing the height (which doesn't change the size of the actual icon) so the amount of space between "Comment" and the comment text is consistent with other fields on the page.

Reply:
We use `rem` in the codebase rather than `em`
```suggestion
                style: { marginLeft: '0.50rem', height: '1.0rem' },
```